tests/testcases/core/domain/services/graphql/connections/PriceConnectionQueriesTest.php

```diff
@@ -0,0 +1,206 @@
+<?php
+
+namespace EventEspresso\tests\testcases\core\domain\services\graphql\connections;
+
+use EE_Dependency_Map;
+use EE_Error;
+use EEM_Price;
+use EEM_Ticket;
+use EventEspresso\tests\testcases\core\domain\services\graphql\GraphQLUnitTestCase;
+use GraphQLRelay\Relay;
+
+class PriceConnectionQueriesTest extends GraphQLUnitTestCase
+{
+    public $subscriber;
+
+    public function setUp()
+    {
+        parent::setUp();
+        if (PHP_VERSION_ID < 70000) {
+            $this->markTestSkipped(
+                'WP GraphQL compatible with PHP 7+ only'
+            );
+            return;
+        }
+
```

Comment:
doesn't need to change, but the `$subscriber` as well as the test skipping could have been added to `GraphQLUnitTestCase` since it appears to be used in every test case class